_posts/2018-12-20-so-long-and-thanks.md

```diff
@@ -0,0 +1,131 @@
+December, the company went through a few big changes. We had (and still have) a shortage
+of product managers which taught me how important they are in keeping the
+software development lifecycle (SDLC) running smoothly especially with scoping out tasks and minimizing
+the friction between stakeholders and engineers. 
+
+I would like to start of by talking about a rather significant change in our engineering process where our
+sprint of three weeks + a few days reserved for what we call slack time - a short period where the team
+works on tech debt and projects that are not of immediate business concern -
+was changed to two weeks with no slack time. However, I feel that this did not pan
+out too well due to these reasons:
+* Meeting durations were not modified to reflect the reduction in sprint duration
+which meant that on the days of those meetings, productivity was hindered due to those
+meetings acting as distractions.
+* In our workflow, a lot of the tasks require collaboration from multiple teams.
+Sometimes the back-and-forth between coworkers working on a feature together takes time 
+and with unavoidable external factors (eg. QA team backlogged, Heroku being down),
+engineers could lose a day or two of productive work. Therefore, the impact of a lost day
+in a two-week sprint compared to a three-week sprint proves to be much more significant.
+* Due to the prior points and the lack of slack time to work on tech debt, the non-immediate
+yet non-trivial tech debt tasks were piling up. Ideally, there was to be a portion of
+time during the sprint dedicated towards tackling technical debt; however, this did not work out
+due to the above reducing the time allotted for prioritized feature work, leaving no room for it.
+
+These are just some of the observations I made during this slightly chaotic time - others
+might have completely different interpretations and thoughts of the matter.
+As of the time of writing this post, we have reverted back to a three-week sprint with a week's
+worth of slack time. From this, I learned that processes can and will change, but they don't necessarily
+need to be permanent and should be critically assessed to see if they work out or not.
+
+# Bite off a Bit More than You Can Chew
+I had absolutely no idea how much I could take on in terms of workload and deliver in a timely manner.
+What if I didn't finish my cards for the sprint? Or what if I break other features with my new one?
+But with time I've come to realize that it wasn't about delivering everything on time or having a perfect
+feature, but rather to extend myself a bit more every time and fail often. During one sprint I had just one card and was hesitant on asking for more (even though they weren't big tasks), but I took another and it went just fine. And although it might not work out everytime, failure in software engineering is the _expected_ outcome, so it's okay. Just as with testing, you'll be constantly disappointed
```

Comment:
I don't know if we were _expecting_ you fail — we expect you to push yourself outside your comfort zone and hope that you are comfortable with the possibility that you MIGHT fail. Give yourself some more credit, Hobin!